saos-webapp/src/main/webapp/WEB-INF/tags/hint.tag

```diff
@@ -0,0 +1,13 @@
+<%@ include file="/WEB-INF/view/common/taglibs.jsp" %>
+<%@ tag display-name="enum" description="Prints localized enum" small-icon=""%>
+
+<%@ attribute name="titleCode" required="true" description="Hint title" rtexprvalue="true" %>
+<%@ attribute name="contentCode" required="true" description="Hint content" rtexprvalue="true" %>
+ 
+<spring:message code="${titleCode}" var="title" />
+<spring:message code="${contentCode}" var="content" />			
+
+<a href="" class="hint" data-toggle="popover" data-trigger="focus" title="${title}" data-content="${content}">
```

Comment:
I used '<a>' here, because this you can select it with keyboard(by 'tabbing' through page).